SDK/MobileCenterAnalytics/Microsoft.Azure.Mobile.Analytics.Windows.Shared/Channel/SessionTracker.cs

```diff
@@ -138,6 +138,9 @@ private void SendStartSessionIfNeeded()
                 _sessions.Remove(_sessions.Keys.Min());
             }
             _sid = Guid.NewGuid();
+#pragma warning disable CS0612 // Type or member is obsolete
+            MobileCenter.CorrelationId = _sid.Value.ToString();
```

Comment:
Ok we discussed this but why analytics does not behave the same as crashes? If crashes comes first then analytics should just reuse that one instead of overwriting. This saves 1 call to watson as a bonus.

Reply:
That makes sense. Can implement.